jobs/io/ndb_io_test.py

```diff
@@ -33,13 +32,14 @@
 
 class NdbIoTests(job_test_utils.PipelinedTestBase):
 
-    def setUp(self):
-        super(NdbIoTests, self).setUp()
-        self.datastoreio_stub = stub_io.DatastoreioStub()
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.all_items_query = (
```

Comment:
Just a note -- I am not sure, but is there an issue with reusing queries? I'm not sure whether performing get() (say) on this object will change its state -- if that's a possibility then maybe you should just construct it locally rather than cache it.

Reply:
Yes, I think it is fine to use the same query again. But anyway I have created separate query everywhere now, since it is not that much places.